docs/components.ipynb

```diff
@@ -0,0 +1,386 @@
+   "source": [
+    "model.choppers = [chopper1]  # Note that model.choppers needs to be a list\n",
+    "model.run()\n",
+    "model.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3155b564-12e1-4d06-a2dd-d376314713f4",
+   "metadata": {},
+   "source": [
+    "As expected, the two openings now create two bursts of neutrons,\n",
+    "separating the wavelengths into two groups.\n",
+    "\n",
+    "If we plot the chopper data,"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "94bc5f13-4e58-4ae4-ba15-86078ab8311d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "chopper1.tofs.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7ce2e25e-e21f-442b-b729-9b1c7a97893b",
+   "metadata": {},
+   "source": [
+    "we see that the chopper sees all the incoming neutrons,\n",
+    "and blocks many of them (gray), only allowing a subset to pass through the openings (blue).\n",
+    "\n",
+    "The detector now sees two peaks in its histogrammed counts:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "90341882-625a-432d-8625-3d0850403f85",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "detector.tofs.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "68e35c62-6072-47b1-8c3e-e610826da008",
+   "metadata": {},
+   "source": [
+    "## Multiple choppers\n",
+    "\n",
+    "It is of course possible to add more than one chopper.\n",
+    "Here we add a second one, further down the beam path,\n",
+    "which splits each groups into two more groups."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ac5f9697-a2a6-45a8-ad99-af6ad1a60be7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "chopper2 = tof.Chopper(\n",
+    "    frequency=20.0 * Hz,\n",
+    "    open=sc.array(\n",
+    "        dims=['cutout'],\n",
+    "        values=[60.0, 100.0, 160.0, 220.0],\n",
+    "        unit='deg',\n",
+    "    ),\n",
+    "    close=sc.array(\n",
+    "        dims=['cutout'],\n",
+    "        values=[90.0, 140.0, 200.0, 260.0],\n",
+    "        unit='deg',\n",
+    "    ),\n",
+    "    phase=60.0 * deg,\n",
+    "    distance=20 * meter,\n",
+    "    name=\"Chopper2\",\n",
+    ")\n",
+    "\n",
+    "model.choppers = [chopper1, chopper2]\n",
+    "model.run()\n",
+    "model.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "62d721c5-3c3e-457c-ab05-60659fccdaaa",
+   "metadata": {},
+   "source": [
+    "The distribution of neutrons that are blocked and pass through the second chopper looks as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ab11bd09-73ba-4bf8-9b52-e503d77f3f5c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "chopper2.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "09d6cbef-f2f3-48db-a696-b4ed912817c8",
+   "metadata": {},
+   "source": [
+    "While the detector now sees 4 peaks"
```

Comment:
"Peaks" may be confusing language, since it may be associated with Bragg peaks.

Reply:
Not sure what else to call it, I also have peaks further up. I don't think 'bursts' is any better?

Reply:
Bunches? Subframes?

Reply:
I kept peaks for now.